src/wkwebkit.cs

```diff
@@ -1025,9 +1030,29 @@ interface WKWebView
 		[NullAllowed, Export ("underPageBackgroundColor", ArgumentSemantic.Copy)]
 		UIColor UnderPageBackgroundColor { get; set; }
 
-		[iOS (15,4), MacCatalyst (15,4), Mac (12,3), NoTV]
+		[iOS (16,0), MacCatalyst (16,0), Mac (13,0), NoTV]
 		[Export ("fullscreenState")]
 		WKFullscreenState FullscreenState { get; }
+
+		[Mac (13,0), iOS (15,5), MacCatalyst(15,5), NoWatch, NoTV]
+		[Export ("minimumViewportInset")]
+		UIEdgeInsets MinimumViewportInset { get; }
+
+		[Mac (13,0), iOS (15,5), MacCatalyst(15,5), NoWatch, NoTV]
+		[Export ("maximumViewportInset")]
+		UIEdgeInsets MaximumViewportInset { get; }
+
+		[Mac (13,0), iOS (15,5), MacCatalyst(15,5), NoWatch, NoTV]
```

Comment:
same smells bad, macOS 13 but iOS 15,5 on a new member? I think they should be bumped to iOS 16

```suggestion
		[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
```

Reply:
https://github.com/xamarin/xamarin-macios/pull/15525#discussion_r922506330